packages/lib/src/spatial-navigation/components/virtualizedList/helpers/getNumberOfItemsVisibleOnScreen.ts

```diff
@@ -0,0 +1,24 @@
+const getMinSizeOfItems = <T>(data: T[], itemSize: number | ((item: T) => number)) => {
+  if (typeof itemSize === 'number') {
+    return itemSize;
+  }
+
+  let minSize = itemSize(data[0]); // Initialize with the size of the first item
+
+  for (let i = 1; i < data.length; i++) {
+    const currentSize = itemSize(data[i]);
+    if (currentSize < minSize) {
+      minSize = currentSize;
+    }
+  }
+
+  return minSize;
+};
+export const getNumberOfItemsVisibleOnScreen = <T>(
+  data: T[],
```

Comment:
déso je débarque un peu car j'ai quasiment jamais relu de PR sur la lib 😬 

Mais `data` c'est pas très clair. C'est des `items` ? Autant que le naming soit cohérent vu qu'on a `itemSize` juste derriere.

Le type générique non plus. est pas clair: `T` ça veut rien dire -> c'est `ItemType` ?

Reply:
(je réponds en anglais comme c'est public)

We used "data" to be as close as possible to the FlatList API. That allows us in the future to have an easier time when working with universal apps :) (FlatList on mobile, SpatialNavigationVirtualizedList on TV, and props will be similar)

Reply:
I'm not against using `data` as the API prop. But I don't think we should use it internally. We can always add an adaptation layer, or at least use `item` everywhere else than inside the exported component

I'm also unresolving the comment because I don't think the type should be called `T`.

Reply:
Indeed this is not resolved!

I totally agree with T that could be named TData for consistency. But I think I disagree with data renaming: it's quite obvious that we're talking about the data props, untransformed, that's given to the list. I think renaming it internally would create add more confusion than clarity 🤔